packages/cli/src/commands/test.js

```diff
@@ -66,6 +66,7 @@ export const handler = async ({
   const args = [
     '--passWithNoTests',
     collectCoverage && '--collectCoverage',
+    '--runInBand'
```

Comment:
Could we only pass this to the api side?

Reply:
Like only add it if the `sides` includes `api`? Yep I think so!

Reply:
Updated, how's that look?